ocropus-visualize-results

```diff
@@ -16,15 +16,20 @@ from scipy.ndimage import interpolation
 
 import ocrolib
 from ocrolib import morph
+from ocrolib.version import VERSION
 
 signal.signal(signal.SIGINT,lambda *args:sys.exit(1))
 
-parser = argparse.ArgumentParser(description = """
+parser = argparse.ArgumentParser(description="""
 Generate HTML for debugging a book directory.
 
 Input: a directory in standard OCRopus book format
 Output: index.html files and thumbnails showing recognition results
 """)
+
+# --version
+parser.add_argument("--version", action="version", version="%(prog)s, v"+VERSION)
```

Comment:
That wouldn't work with `distutils.version.StrictVersion` and also doesn't follow PEP440, and well, why do you need the `v` anyway?

Reply:
Because that is incompatible with pypi and python best practices. I'm doing it here mostly to make it easier to pattern match, like in the run-test-ci script.